tests/std/tests/P1614R2_spaceship/test.cpp

```diff
@@ -484,6 +485,22 @@ void ordering_test_cases() {
 
         spaceship_test<std::strong_ordering>(c_mem[0], c_mem[0], c_mem[1]);
     }
+    { // variant
+        using V = std::variant<int, long>;

```

Comment:
There's nothing testing the `valueless_by_exception()` state, which is specified to be less than all other states (and equal to itself). I expect that the common machinery gets this right, but we should have test coverage for it.